test/contentful-config.ts

```diff
@@ -0,0 +1,35 @@
+import { writeFile, stat } from 'fs/promises'
+
+const configFile = process.env.CONTENTFUL_CONFIG_FILE
+
+export async function initConfig() {
+  if (!configFile)
+    throw Error(
+      `Please specify .contentfulrc.json path
+      Set env variable CONTENTFUL_CONFIG_FILE`
+    )
+
+  try {
+    await stat(configFile)
+  } catch (e) {
+    return writeFile(
+      configFile,
+      JSON.stringify(
+        { managementToken: process.env.CLI_E2E_CMA_TOKEN },
+        null,
+        4
+      )
+    )
+  }
+
+  const configParams = require(configFile)
```

Comment:
One thing I forgot: using `require` here instead of `readFile` feels funky to me, but I looked it up and other than caching differences it seems it's fine 👍